View/Elements/cou.ctp

```diff
@@ -20,81 +20,133 @@
                         <?php
                         $attrs = array();
                         $attrs['value'] = RciamStatsViewerDateEnum::yearly;
                         $attrs['empty'] = false;
-
                         print $this->Form->select(
-                            'date' . ucfirst($prefix) . 'Select',
-                            RciamStatsViewerDateEnum::type,
-                            $attrs
+                          'date' . ucfirst($prefix) . 'Select',
+                          RciamStatsViewerDateEnum::type,
+                          $attrs
                         );
 
                         if ($this->Form->isFieldError('date' . ucfirst($prefix) . 'Select')) {
-                            print $this->Form->error('date' . ucfirst($prefix) . 'Select');
+                          print $this->Form->error('date' . ucfirst($prefix) . 'Select');
                         }
                         ?>
+                      </div>
+                  </div>
+                  <div class="col-lg-9">
+                      <div class="columnChart" id="<?php
+                      print Inflector::pluralize($prefix); ?>ChartDetail"></div>
+                  </div>
+                  <div class="col-lg-3">
+                      <ul class="<?php
+                      print $prefix; ?>Names columnList">
+                      </ul>
+                  </div>
+              </div>
+          </div>
+          <div class="box" data-type="<?php print $prefix ?>">
+              <div class="box-header with-border">
+                  <h3 class="box-title"><?php print _txt('pl.rciamstatsviewer.' . $prefix . '.pl'); ?> </h3>
+              </div>
+              <!-- /.box-header -->
+              <div class="box-body dataTableWithFilter">
+                <?php
+                if ($vv_permissions['registered']) : ?>
+                    <div class="dataTableDateFilter bg-box-silver">
+                        From: &nbsp;
+                        <input type="text" id="<?php print $prefix ?>DateFrom" name="<?php print $prefix ?>DateFrom" data-provide="datepicker"/>
+                        &nbsp;&nbsp;&nbsp;To: &nbsp;
+                        <input type="text" id="<?php print $prefix ?>DateTo" name="<?php print $prefix ?>DateTo" data-provide="datepicker"/>
+                      <?php if ($prefix == 'registered' || $prefix == 'cou') : ?>
+                          <div class="btn-group">
+                              &nbsp;
+                              <button type="button" class="btn btn-default dropdown-toggle filter-button"
+                                            data-toggle="dropdown">
+                                  Filter <span class="caret"></span>
+                              </button>
+                              <ul class="dropdown-menu">
+                                  <li><a href="#" onclick="return false;" class="groupDataByDate" data-value="daily">Daily
```

Comment:
Implement this with foreach. The values should be enumerators

Reply:
Done, I have put it to an element also